Core/Helper.php

```diff
@@ -290,4 +299,26 @@ public static function isThisModule($sModuleId)
     {
         return $sModuleId === self::MODULE_ID;
     }
+
+    /**
+     * @return array
+     *
+     * @since 1.0.0
+     */
+    public static function getShopInfoFields()
+    {
+        $shopId = Registry::getConfig()->getShopId();
+        $shop = oxNew(Shop::class);
+        $shop->load($shopId);
+
+        $oModule = oxNew(Module::class);
+        $oModule->load(Helper::MODULE_ID);
+
+        return [self::SHOP_SYSTEM_KEY => "OXID",
```

Comment:
Add to new line and maybe put the OXID string into constant, as we have "wdoxidee" module ID?

Reply:
👍